src/store/GlobalState.js

```diff
@@ -0,0 +1,155 @@
+import { createContext, useEffect, useReducer } from 'react';
+import { collection, doc, addDoc, deleteDoc, updateDoc, query, onSnapshot } from 'firebase/firestore';
+
+import db from '../firebase';
+import appReducer from './appReducer';
+import { getCurrentMonth, getUKFormattedDate } from '../utils/helpers';
+
+const currentMonth = getCurrentMonth();
+
+const initialState = {
+  expenses: [],
+  selectedMonthExpenses: [],
```

Comment:
the name suggests that this returns the expenses for the selected month. shouldn't this be a function/computed property and not a state?

Reply:
Yeah I looked into this, and it seems you're right. Good catch :thumbsup: